packages/cli/__tests__/zosjobs/__integration__/submit/local-file/cli.zos-jobs.submit.local-file.integration.test.ts

```diff
@@ -40,7 +40,6 @@ describe("zos-jobs submit local-file command", () => {
         const response = runCliScript(__dirname + "/__scripts__/submit_invalid_local_file.sh",
             TEST_ENVIRONMENT);
         expect(response.status).toBe(1);
-        expect(response.stderr.toString().toLowerCase()).toContain("error");
```

Comment:
Looking into why this fails and the snapshot changed, will post an update if I figure it out 🙂